crates/bevy_text/src/text2d.rs

```diff
@@ -123,7 +134,7 @@ pub fn text2d_system(
     mut text_pipeline: ResMut<DefaultTextPipeline>,
     mut text_queries: QuerySet<(
         QueryState<Entity, (With<Text2dSize>, Changed<Text>)>,
-        QueryState<(&Text, &mut Text2dSize), With<Text2dSize>>,
+        QueryState<(&Text, &Text2dBounds, &mut Text2dSize), With<Text2dSize>>,
```

Comment:
We can use an `Option<Text2dBounds>` here to be robust in the cases where this component isn't set.

Reply:
OK, done now.